excelerator/imports/iderdown.js

```diff
@@ -131,9 +131,11 @@ export function processIdJob(job, outputStream) {
 
     var startTime = new Date(); //not actually the start, but close enough for the timeout purposes
 
+
+    //Commenting this block out to enable corsswalks
     var inHierarchy = outputType.withinTypes.indexOf(params.filterTypeUri) !== -1;
```

Comment:
So long as we all agree this sucks as a way of checking same dataset, I will let it slide (pretty sure I use something like this somewhere). Should probably be in a function we can fix later.